src/Our.Umbraco.Ditto/ComponentModel/ValueResolvers/AppSettingValueResolver.cs

```diff
@@ -28,12 +28,21 @@ public class AppSettingValueResolver : DittoValueResolver<AppSettingAttribute>
         /// </returns>
         public override object ResolveValue(ITypeDescriptorContext context, AppSettingAttribute attribute, CultureInfo culture)
         {
-            if (string.IsNullOrWhiteSpace(attribute.AppSettingKey))
+            // default to attribute value
+            string appSettingKey = attribute.AppSettingKey;
```

Comment:
Thanks @mattbrailsford @Hendy ... sure, null-reference checking is fine, (I'm usually an over-defensive coder :bowtie:)

@Hendy Did you want to amend the code? I'm happy to merge in whenever (we can pick up small tidy-ups later)